lib/ex_doc.ex

```diff
@@ -1,3 +1,20 @@
+defmodule ExDoc.Default do
```

Comment:
We don't need a new module, it can just be in `ExDoc`. Don't complicate the feature. :)